app/main.go

```diff
@@ -34,44 +34,32 @@ func main() {
 		log.Fatal("Failed to load the server config: " + err.Error())
 	}
 
-	conf.LoadAppConfig()
+	// Load the App config
+	err = conf.LoadAppConfig()
+	if err != nil {
+		log.Fatal("Failed to load the app config: " + err.Error())
+	}
+
 	conf.StartConfigManager()
 
+	// setup the router and the api
 	router := mux.NewRouter()
-
-	// Init the meta api
 	api.InitMetaHandlers(router, "api")
 
 	// check to see if we have a defined running config
 	// If not we are only going to boot the setup apis, otherwise we will start the app
 	if conf.AppConf.RunningNavVersion == "" {
 
-		//log.Println("App config undetected :: creating mock config, initializing setup handlers")
-
-		//appConfig, err := conf.MockAppConfig()
-		//if err != nil {
-		//	log.Fatal("Failed to create the mock config: " + err.Error())
-		//} else {
-		//	log.Println("appConfig", appConfig)
-		//}
-
+		log.Println("No App Config starting the setup api")
 		setupapi.InitSetupHandlers(router, "api")
 
```

Comment:
random question; are pure lowercase variables a style of GoLang? something like this perhaps? `lowercasevar.UpperCamelCaseMethod()`

Reply:
@matt-auckland yeah we've been using that `lowercase` style throughout for vars that are local to functions, but the general recommendation is that var names in Go should be short rather than long. This is especially true for local variables with limited scope. so you'll see lots of `aa` or `x` type convention in projects... which can be a bit unreadable at times so we've been lengthening ours a bit for clarity. We've also been adhering to the `lowerCasePrivateFunction()` and `UpperCasePublicFunction` convention as well 👍